docs/guides/verify-shapella-upgrade-manual.md

```diff
@@ -0,0 +1,530 @@
+# Shapella mainnet parameters
+
+## Mainnet addresses
+
+```python
+lidoLocator = 0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb
+legacyOracle = 0x442af784A788A5bd6F42A01Ebe9F287a871243fb  # was `LidoOracle` before
+
+# See https://docs.lido.fi/deployed-contracts/
+# Old implementation was `Withdrawals Manager Stub`
```

Comment:
maybe add 'Represents an address on Execution Layer corresponding to the Lido `0x01`-type withdrawal credentials (see below)'